src/ray/raylet/scheduling/cluster_task_manager.h

```diff
@@ -79,6 +83,24 @@ class ClusterTaskManager {
   /// \param readyIds: The tasks which are now ready to be dispatched.
   void TasksUnblocked(const std::vector<TaskID> ready_ids);
 
+  /// (Step 5) Call once a task finishes.
```

Comment:
What's step 5?

Reply:
See line 38?

Reply:
Maybe clarify that this should be called when a worker is returned.

I might also call it `HandleTaskFinished`.